tests/models/whisper/test_modeling_whisper.py

```diff
@@ -357,9 +357,26 @@ def _get_input_ids_and_config(self):
 
         return config, input_ids, None, max_length
 
-    # not implemented currently
     def test_inputs_embeds(self):
-        pass
+        config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
+
+        for model_class in self.all_model_classes:
+            model = model_class(config)
+            model.to(torch_device)
+            model.eval()
+
+            inputs = copy.deepcopy(self._prepare_for_class(inputs_dict, model_class))
+
+            encoder_input_ids = inputs["input_features"]
+            decoder_input_ids = inputs.get("decoder_input_ids", encoder_input_ids)
+            inputs.pop("decoder_input_ids", None)
```

Comment:
```suggestion
decoder_input_ids = inputs.pop("decoder_input_ids", None)
```
Does this also work, feels a bit strange to default the `decoder_input_ids` to encoder's no? (small nit)